volatility/framework/layers/vmware.py

```diff
@@ -5,12 +5,16 @@
 import struct
 from typing import Any, Dict, List, Optional
 
-from volatility.framework import interfaces, constants
+from volatility.framework import interfaces, constants, exceptions
 from volatility.framework.configuration import requirements
 from volatility.framework.layers import physical, segmented, resources
 from volatility.framework.symbols import native
 
 
+class VmwareFormatException(exceptions.LayerException):
+    """Thrown when an error occurs with the underlying Crash file format."""
```

Comment:
This says Crash file format, but should say VMware file format.

Reply:
Good spot, thanks, I've fixed this now.  5:)